static/app/views/dashboards/manage/index.tsx

```diff
@@ -241,6 +241,21 @@ class ManageDashboards extends AsyncView<Props, State> {
     );
     this.onDashboardsChange();
     addSuccessMessage(`${dashboard.title} dashboard template successfully added.`);
+    setTimeout(() => {
+      this.loadDashboard(dashboard.title);
+    }, 500);
```

Comment:
We can remove the call to `this.onDashboardsChange()` because if we're redirecting to the dashboard now, there isn't actually much benefit to reloading the dashboards briefly

That also means we can remove the surrounding `setTimeout` because we should just redirect once we know the dashboard's been saved

Reply:
Got it! I'll fix this. Thank you for the review!